src/lightning_app/utilities/app_helpers.py

```diff
@@ -184,11 +184,20 @@ def render_non_authorized(self):
 
 
 def target_fn():
-    from streamlit.server.server import Server
```

Comment:
I think we should have a comment that explains in which version of streamlit this changed. So that we can update the code should the minimum version ever be bumped

Reply:
Done :smiley: